wayshot/src/wayshot.rs

```diff
@@ -33,29 +31,75 @@ where
 }
 
 fn main() -> Result<()> {
+    // cli args
     let cli = cli::Cli::parse();
+
+    // config path
+    let config_path = dirs::config_local_dir()
+        .map(|path| path.join("wayshot").join("config.toml"))
+        .unwrap_or_default();
+    let config_path = cli.config.unwrap_or(config_path);
+
+    // config
+    let config = Config::load(&config_path).unwrap_or_default();
+    let log = config.log.unwrap_or_default();
+    let screenshot = config.screenshot.unwrap_or_default();
```

Comment:
"screenshot" sounds very vague when reading a screenshot apps codebase. Maybe something better can be used.

I understand if these nits are disturbing to deal with but I hope you see where I'm coming from, we keep the code standards high and unambiguous to help our contributors out ( I hope it was a nice experience for you ❤️ )

Reply:
Good point. I was thinking about that section of config as a basic properties. Maybe `basic` or `base`, something like that would do?

> I understand if these nits are disturbing to deal with but I hope you see where I'm coming from, we keep the code standards high and unambiguous to help our contributors out ( I hope it was a nice experience for you ❤️ )

Don't have any objections, just want to implement a clean and useful feature for a great software 😊

Reply:
I think base sounds better.

Reply:
Done